README.md

```diff
@@ -1 +1,207 @@
+
+#### To enable Logs collection
+
+Download all the yaml files from [this dir](/logan/kubernetes-resources/logs-collection/).
+These yaml files needs to be applied using kubectl to create the necessary resources that enables the logs collection into Logging Analytics through a Fluentd based DaemonSet.
+
+##### configmap-docker.yaml | configmap-cri.yaml
+
+- This file contains the necessary out of the box fluentd configuration to collect Kubernetes System/Service Logs, Linux System Logs and Application Pod/Container Logs. 
+- Some log locations may differ for Kubernetes clusters other than OKE, EKS and may need modifications accordingly. 
+A comprehensive out of the box covering typical kubernetes clusters will be available soon.
+- Use configmap-docker.yaml for Kubernetes clusters based off Docker runtime (e.g., OKE < 1.20) and configmap-cri.yaml for Kubernetes clusters based off CRI-O.
+- Inline comments are available in the file for each of the source/filter/match blocks for easy reference for making any changes to the configuration.
+- Refer [this](https://docs.oracle.com/en/learn/oci_logging_analytics_fluentd/) to learn about each of the Logging Analytics Fluentd Output plugin configuration parameters.
+- *Note*: A generic source with time only parser is defined/configured for collecting all application pod logs from /var/log/containers/ out of the box. 
+          It is recommended to define and use a LogSource/LogParser at Logging Analytics for a given log type and then modify the configuration accordingly.
+          When adding a configuration (Source, Filter section) for any new container log, also exclude the log path from generic log collection, 
+            by adding the log path to *exclude_path* field in *in_tail_containerlogs* source block. This is to avoid the duplicate collection of logs through generic log collection.
+
+##### fluentd-daemonset.yaml
+
+- This file has all the necessary resources to deploy to run the Fluentd docker image as Daemonset.
+- Inline comments are available in the file describing each of the fields/sections. 
+- Make sure to replace the fields with actual values before deploying. 
+- At minimum, <IMAGE_URL>, <OCI_LOGGING_ANALYTICS_LOG_GROUP_ID>, <OCI_TENANCY_NAMESPACE> needs to be updated. 
+- It is recommended to update, <KUBERNETES_CLUSTER_OCID>,<KUBERNETES_CLUSTER_NAME> too tag all the logs with corresponding Kubernetes cluster at Logging Analytics. 
+
+##### secrets.yaml (Optional)
+
+- At present, InstancePrincipal and OCI Config File (UserPrincipal) based Auth/AuthZ are supported for Fluentd to talk to OCI Logging Analytics APIs. 
+- We recommend to use InstancePrincipal based AuthZ for OKE and all clusters which are running on OCI VMs and that is the default auth type configured. 
+- Applying this file is not required when using InstancePrincipal based auth type.
+- You need to modify this file to fill out the values under config section with appropriate values.
```

Comment:
When config file based Authz is used, modify this file to fill out the values under config section with appropriate values.